async_mock/coroutine.py

```diff
@@ -16,6 +20,7 @@ def __init__(self, loop, returnSequence, block:asyncio.Event):
         self.__returnSequenceLen = len(self.__returnSequence)
         self.__block = block
         self.__mock = self.__createMock()
+        self.__blocking_dict = CoroutineMock.__blocking_dict
```

Comment:
Are you intending __blocking_dict to be a singleton for the class? Because this setup has all instances sharing one object.  Also, this line in the constructor is totally unnecessary since the instance variable and the static class variable share a name.  You can omit this line and still call self.__blocking_dict to have the same effect since static class variables can be accessed through self.